src/main/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -0,0 +1 @@
+io.pinecone.AfterAllTestsListener
```

Comment:
Apparently, this is one way of registering a `TestExecutionListener` with Junit. 

Tbh I went through a lot of trial and error, but I found guidance for this in a few places: https://stackoverflow.com/a/63774284

Reply:
Odd, I wouldn't have guessed this. 

Can you move the file to META-INF in test folder? `test/main/resources/META-INF/etc...`

Reply:
Good call! I had tried adding a `/META-INFO/services/` folder to the `test` and `integration` modules, but I didn't delete things out of `main` which is why it wasn't registering properly.

I've managed to get things working properly by adding `/resources/META-INFO/services/` to the `integration` folder and removing it from elsewhere. 🎉